### PR TITLE
fix e2e prometheus, operator-group, 4.11 errors

### PR DESCRIPTION
**Pull request by PratikMahajan**

### CODE REVIEW

**Comment by PratikMahajan** [comment]
/retest 
imagepullbackoff for operator

**Comment by PratikMahajan** [comment]
/retest 
cluster creation failure

File: hack/e2e.sh [hunk 1]
```diff
@@ -127,8 +129,8 @@ backoff oc -n openshift-update-service wait --for=condition=Ready pod -l app=e2e
 }
 
 # Expose services
-PE_URL=$(oc get route e2e-policy-engine-route -o jsonpath='{.spec.host}')
-export GRAPH_URL="https://${PE_URL}/api/upgrades_info/graph"
+PE_URL=$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}' updateservice "e2e")
```
Comment (wking) [inline hack/e2e.sh]: could be a race here, if the controller hasn't populated the property yet.  From [our docs][1]:

> Use the following command to obtain a policy engine route:
>
> ```console
> $ while sleep 1; do POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph{"\n"}' updateservice "${NAME}")"; SCHEME="${POLICY_ENGINE_GRAPH_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done
> ```
>
> You might need to poll until the command succeeds.

[1]: https://docs.openshift.com/container-platform/4.11/updating/updating-restricted-network-cluster.html#update-service-create-service-cli_updating-restricted-network-cluster

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**